src/Framework/test/SharedFxTests.cs

```diff
@@ -24,6 +24,33 @@ public SharedFxTests(ITestOutputHelper output)
             _expectedRid = TestData.GetSharedFxRuntimeIdentifier();
         }
 
+        [Fact]
+        public void SharedFrameworkContainsExpectedFiles()
+        {
+            var actualAssemblies = Directory.GetFiles(TestData.GetTestDataValue("RuntimeAssetsOutputPath"), "*.dll")
+                .Select(Path.GetFileNameWithoutExtension)
+                .ToHashSet();
+            var expectedAssemblies = TestData.GetSharedFxDependencies()
+                .Split(';', StringSplitOptions.RemoveEmptyEntries)
+                .ToHashSet();
+
+            _output.WriteLine("==== actual assemblies ====");
+            _output.WriteLine(string.Join('\n', actualAssemblies));
```

Comment:
Just curious - for the sake of printing these out, would it make sense to sort them by some arbitrary string sort? That would make the debug output easier for humans to read (I realize it doesn't technically matter).

Reply:
Yeah, we could order by name. I'll add that when I merge this back to master (i have green CI checks right now, and don't want to wait 90 minutes to fix this up.)